app/controllers/games_controller.rb

```diff
@@ -1,17 +1,20 @@
 class GamesController < ApplicationController
-  def new
-    config = GameConfig.find(params[:game_config_id])
+  def create
+    config = GameConfig.find_by(id: params[:game_config_id])
 
-    config.update(locked: true)
+    config.update!(locked: true)
```

Comment:
This would mean I have to duplicate the card data on each game, instead of it being a lookup...

As card editing (in theory) is lower throughput than games this seems wasteful..

Reply:
This also means that starting the same game twice doesn't result in playing different games... That being said I could see an argue for having a config in develop mode that would allow editing during game play.... let me think about that a bit as it might be possible